bigbluebutton-html5/private/locales/pt_BR.json

```diff
@@ -304,6 +304,19 @@
     "app.video.videoMenuDesc": "Abra o menu de vídeo",
     "app.video.chromeExtensionError": "Você deve instalar o seguinte:",
     "app.video.chromeExtensionErrorLink": "esta extensão do Chrome",
+    "app.video.stats.title": "Estatísticas da Conexão",
+    "app.video.stats.packetsReceived": "Pacotes recebidos",
+    "app.video.stats.packetsSent": "Pacotes enviados",
+    "app.video.stats.packetsLost": "Pacotes perdidos",
```

Comment:
We typically pull translations from Transifex, I am not sure if adding these manually will break pulling from transifex. cc @KDSBrowne